macOS/LocalPackages/NewTabPage/Sources/NewTabPage/RMF/NewTabPageDataModel+RMF.swift

```diff
@@ -133,6 +138,16 @@ extension NewTabPageDataModel {
                 self = .subscription
             case .aiChat:
                 self = .duckai
+            case .pir:
+                self = .pir
+            case .radar:
+                self = .radar
+            case .radarNew:
```

Comment:
I tested `radarNew` on macOS and it didn’t resolve to anything (shows a broken icon). Since `radar` is using the new icon, how would you feel about just removing `radarNew` completely on macOS? (I believe `radar` was previously just added on iOS)